app/assets/v2/js/pages/dashboard.js

```diff
@@ -324,11 +324,11 @@ var process_stats = function(results) {
       break;
     case 1:
       matchesEl.html(num + gettext(' Matching Result'));
-      fundingInfoEl.html("<span id='modifiers'>Funded Issue</span><span id='stats' class='font-body'>(" + stats + ')</span>');
+      fundingInfoEl.html("<span id='modifiers'>Funded Issue</span><span id='stats' class='font-caption'>(" + stats + ')</span>');
```

Comment:
I know it's not your fault but I think `')</span>'` to `")</span>"` keeps the uniformity.

Reply:
Nice catch